main/py-pillow/APKBUILD

```diff
@@ -23,6 +23,12 @@ build() {
 	python3 setup.py build
 }
 
+build() {
+	cd "$builddir"
+	python2 setup.py test
+	python3 setup.py test
```

Comment:
why tests running in `build()`?

Reply:
Thanks for the hint, I was already wondering about the strange output.